prince-of-versions/src/test/java/co/infinum/princeofversions/JsonConfigurationParserTest.java

```diff
@@ -41,28 +34,28 @@ public void checkEmptyJsonToStringMap() throws Throwable {
     @Test
     public void checkJsonToStringMap() throws Throwable {
         assertThat(
-            parser.jsonObjectToMap(new JSONObject(ResourceUtils.readFromFile("json_obj_string.json")))
+                parser.jsonObjectToMap(new JSONObject(ResourceUtils.readFromFile("json_obj_string.json")))
```

Comment:
Why double indenting?

Reply:
I have no idea, I just reformatted the file. I'll check my code style config.